bats/tests/containers/factory-reset.bats

```diff
@@ -86,57 +254,106 @@ check_installation() {
         # this one only exists after an update has been downloaded
         # ~/Library/Application Support/Caches/rancher-desktop-updater
     fi
+
+    if is_windows; then
+        delete_dir+=("$PATH_LOGS" "$PATH_DISTRO" "$PATH_DISTRO_DATA")
+    fi
+
     for dir in "${delete_dir[@]}"; do
         echo "$assert that $dir does not exist"
-        ${assert}_not_exists "$dir"
+        "${assert}"_not_exists "$dir"
     done
+}
 
+check_docker_symlinks() {
+    local assert=assert
+    local refute=refute
+
+    if [ "${1-}" == "before" ]; then
+        assert=refute
+        refute=assert
+    fi
     # Check if docker-X symlinks were deleted
     for dfile in docker-buildx docker-compose; do
         run readlink "$HOME/.docker/cli-plugins/$dfile"
-        ${refute}_output "$HOME/.rd/bin/$dfile"
+        "${refute}"_output "$HOME/.rd/bin/$dfile"
```

Comment:
I don't care too much, but I don't think adding the quotes improves readability.

We already run shellcheck over the files, and it notices that the only values for `$refute` are `assert` or `refute`, and neither contain spaces, so it doesn't require you to put the variable in quotes.